wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -97,49 +200,58 @@ int tsip_hw_lock()
 */
 void tsip_hw_unlock( void )
 {
-    WOLFSSL_MSG("enter tsip_hw_unlock");
-    /* unlock hw engine for next use */
+    /* WOLFSSL_MSG("enter tsip_hw_unlock"); */
     tsip_CryptHwMutexUnLock(&tsip_mutex);
-    WOLFSSL_MSG("leave tsip_hw_unlock");
+    /* WOLFSSL_MSG("leave tsip_hw_unlock");*/
 }
+
 /* check if tsip tls functions can be used for the cipher      */
 /* cipher0 : in the some cipher suite,                         */
 /*           first byte becomes greater than 0, otherwise 0x00 */
 /* side    : CLIENT END or SEVER END                           */
 int tsip_useable(const struct WOLFSSL *ssl)
 {
+    WOLFSSL_MSG(">> tsip_useable"); 
     byte cipher0;
     byte cipher;
     byte side;
     
     /* sanity check */
-    if (ssl == NULL)
+    if (ssl == NULL){
+        WOLFSSL_MSG("<< tsip_useable: No");
         return BAD_FUNC_ARG;
-    
+    }
     /* when rsa key index == NULL, tsip isn't used for cert verification. */
     /* in the case, we cannot use TSIP.                                   */
-    if (!ssl->peerTsipEncRsaKeyIndex)
+    if (!ssl->peerTsipEncRsaKeyIndex){
+        WOLFSSL_MSG("<< tsip_useable: No");
```

Comment:
In the case where TSIP is not usable, should we include a reason in the debug message?  This would let users know why TSIP was not used.

Reply:
This function is called many times and it makes important judgements but does not say any reason of them. To make logs helpful, it should leave an error code into log. On the other hand, it might be true too much internal detail should not be exposed to users. Therefore, I put "Yes/No" logs.  

Whatever the result this function returns, trailing functions will leave meaningful log. So I will remove debug messages.